Source/NSMenuView.m

```diff
@@ -1826,18 +1827,24 @@ - (BOOL) _trackWithEvent: (NSEvent*)event
         }
 
       do
-	{
-	  event = [NSApp nextEventMatchingMask: eventMask
-				     untilDate: theDistantFuture
-					inMode: NSEventTrackingRunLoopMode
-				       dequeue: YES];
-	  type = [event type];
-	  if (type == NSAppKitDefined)
-	    {
-	      [[event window] sendEvent: event];
-	    }
-	}
-      while (type == NSAppKitDefined);
+      {
```

Comment:
Could you please stick with the current GNUstep indentation. I know that it is annoying to get it right, but we agreed on it a long time ago and now we should keep the code that way.

Reply:
Sorry I have fixed this now.